cmd/tailscale/cli/serve_v2.go

```diff
@@ -45,16 +45,13 @@ a partial URL (e.g., localhost:3000), or a full URL including a path (e.g., http
 
 EXAMPLES
   - Expose an HTTP server running at 127.0.0.1:3000 in the foreground:
-    $ tailscale %s 3000
+    $ tailscale %[1]s 3000
 
   - Expose an HTTP server running at 127.0.0.1:3000 in the background:
-    $ tailscale %s --bg 3000
-
-  - Expose an HTTPS server with a valid certificate at https://localhost:8443
```

Comment:
Removed this for now [[ref](https://github.com/tailscale/tailscale/issues/9923#issuecomment-1775687089)]